reqs/models.py

```diff
@@ -69,6 +69,7 @@ class Meta:
     sunset = models.DateField(blank=True, null=True)
     policy_status = models.CharField(max_length=256, blank=True)
     document_source = models.FileField(blank=True)
+    nonpublic = models.NullBooleanField()
```

Comment:
Do we really want to allow this boolean to be null (i.e. three-value logic)? I'd prefer if we picked a default instead:
* `False` is pseudo implicit given that we only ever use this logic to `exclude`
* `True` might make sense as a safety for OMB (forcing them to actively "publish")

Either way, we'd want a migration script which sets all existing policies to `False`